### PR TITLE
Add SLE Micro 5.2

### PR DESCRIPTION
**Pull request by nodeg**
## What does this PR change?

This will

- add SLE Micro 5.2 support

### CODE REVIEW

File: modules/cucumber_testsuite/main.tf [hunk 1]
```diff
@@ -115,7 +115,7 @@ module "suse-client" {
   quantity = contains(local.hosts, "suse-client") ? 1 : 0
   base_configuration = module.base.configuration
   product_version    = var.product_version
-  image              = lookup(local.images, "suse-client", "sles15sp2o")
+  image              = lookup(local.images, "suse-client", "sles15sp4o")
```
Comment (srbarrios) [inline modules/cucumber_testsuite/main.tf:118]: Better don't mix PRs.
This PR should only have changes to support SLE Micro.
Move other stuff in another PR please.

**Review by srbarrios — APPROVED** [review]

**Comment by srbarrios** [comment]
We might need to also include one VM for SSH Salt Minion, let's wait for Abid's reply.

File: modules/cucumber_testsuite/main.tf (hunk 1 above)
Comment (nodeg) [reply modules/cucumber_testsuite/main.tf:118]: I can do that. I used explicitly 2 commits to separate the changes.

**Review by srbarrios — APPROVED** [review]
